src/components/OptionsSelector/BaseOptionsSelector.js

```diff
@@ -60,7 +61,14 @@ class BaseOptionsSelector extends Component {
                     return;
                 }
 
-                this.selectRow(focusedOption);
+                // We are handeling multiple Enter events to address the issue in https://github.com/Expensify/App/issues/18265
```

Comment:
I don't think we need to mention the general requirement of the PR, since the git blame will show the PR if required